gateway/src/config/mod.rs

```diff
@@ -159,6 +159,7 @@ impl Config {
         self
     }
 
+    #[allow(unused)]
```

Comment:
I guess since it was only used for upgrades, lets remove it? We can always add it back in if we need it